keps/0007-kic-performance-test.md

```diff
@@ -0,0 +1,557 @@
+---
+title: KIC Performance Test Solution
+status: implementable
```

Comment:
We need to start this out in a `provisional` status, as this will indicate that we're still making some core decisions about the proposal and nobody should be starting an implementation (save perhaps a spike or prototype to support the KEP) yet:

```suggestion
status: provisional
```